spec/bot_spec.rb

```diff
@@ -92,7 +92,7 @@
         let(:comment) { "foo\nbar\nbaz QA: this\nis\na\nmultiline\ncomment" }
 
         it "uses all lines" do
-          expect(Jira::Comment).to receive(:create).with("LIEF-123", "this\nis\na\nmultiline\ncomment")
+          expect(Jira::Comment).to receive(:create).with("LIEF-123", "foo\nbar\nbaz QA: this\nis\na\nmultiline\ncomment")
```

Comment:
This was by-design... so that a github comment can include messages that are meant for the reviewer AND for QA. E.g.

```
Doing this becasue X and Y

QA: make sure this doesn't break Z
```